status.sh

```
@@ -0,0 +1,151 @@
+    echo "   • Memory: $(free -h | grep '^Mem:' | awk '{print $2}') total, $(free -h | grep '^Mem:' | awk '{print $3}') used"
+    echo "   • Disk: $(df -h . | tail -1 | awk '{print $2}') total, $(df -h . | tail -1 | awk '{print $3}') used"
+    echo "   • Docker: $(docker --version 2>/dev/null | cut -d' ' -f3 | sed 's/,//' || echo 'Not installed')"
+    echo "   • Python: $(python3 --version 2>/dev/null | cut -d' ' -f2 || echo 'Not installed')"
+}
+
+# Функция для получения информации о Docker контейнерах
+get_docker_info() {
+    print_colored "🐳 Docker Containers:" "$BLUE"
+    
+    if command -v docker &> /dev/null; then
+        local userbot_containers=$(docker ps -a --filter "name=userbot_" --format "table {{.Names}}\t{{.Status}}\t{{.Ports}}" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)